src/server.ts

```diff
@@ -476,6 +523,43 @@ INNER JOIN ntee_organization_type n_o_t
               );
             },
           },
+          addPdf: {
+            type: pdfType,
+            args: { input: { type: new GraphQLNonNull(pdfInputType) } },
+            resolve: async (source, { input }, context) => {
+              const authenticatedUser = await getUserFromToken(
+                context.token,
+                context.oauthClient,
+                db
+              );
+
+              if (!authenticatedUser) throw new Error('not authenticated');
+
+              const organization = await db.Organization.find({
+                where: { uuid: input.organization },
+              });
+
+              if (!organization) throw new Error('missing organization');
+
+              const user = input.user
```

Comment:
QQ, so we can specify a user in the form/input, and then we fall back to the authenticated user?

Reply:
ooh yeah i think this should change -- we should be able to add a pdf without assigning a user right away.